### ISSUE TITLE
Missing links in documentation 

### ISSUE THREAD

**Issue by haahh** [boostorg/compute#652]
In [documentation on boost.org](http://www.boost.org/doc/libs/1_61_0/libs/compute/doc/html/boost_compute/reference.html#boost_compute.reference.api_overview) links to APIs of core classes are not there, however, everything is fine [here](http://boostorg.github.io/compute/boost_compute/reference.html#boost_compute.reference.api_overview) on GitHub.

**Comment by keryell** [comment]
Yes and I wonder if it is not related to https://github.com/boostorg/compute/issues/621

**Comment by keryell** [comment]
Is this considered as solved ?

**Comment by ameyanator** [comment]
Is this issue still open?

**Comment by keryell** [comment]
Is there anyone with Boost build infrastructure set up to try, please?
